src/dotnet-install.sh

```diff
@@ -1119,93 +1121,219 @@ get_download_link_from_aka_ms() {
     fi
 }
 
-calculate_vars() {
-    eval $invocation
-    valid_legacy_download_link=true
+get_feeds_to_use()
+{
+    feeds=(
+    "https://dotnetcli.azureedge.net/dotnet"
+    "https://dotnetbuilds.azureedge.net/public"
+    )
 
-    #normalize input variables
-    normalized_architecture="$(get_normalized_architecture_from_architecture "$architecture")"
-    say_verbose "Normalized architecture: '$normalized_architecture'."
-    normalized_os="$(get_normalized_os "$user_defined_os")"
-    say_verbose "Normalized OS: '$normalized_os'."
-    normalized_quality="$(get_normalized_quality "$quality")"
-    say_verbose "Normalized quality: '$normalized_quality'."
-    normalized_channel="$(get_normalized_channel "$channel")"
-    say_verbose "Normalized channel: '$normalized_channel'."
-    normalized_product="$(get_normalized_product "$runtime")"
-    say_verbose "Normalized product: '$normalized_product'."
+    if [[ -n "$azure_feed" && "$azure_feed" != "" ]]; then
```

Comment:
nit: The second part of the condition seems redundant. According to https://linux.die.net/man/1/test -n tests whether _the length of STRING is nonzero_.